src/content/docs/identityserver/upgrades/identityserver4-to-duende-identityserver-v7.mdx

```diff
@@ -0,0 +1,487 @@
+---
+title: "IdentityServer4 to Duende IdentityServer v7.2"
+sidebar:
+  order: 138
+  label: IdentityServer4 → v7.2
+---
+
+import { Badge } from "@astrojs/starlight/components";
```

Comment:
Badge is an unused import. You can remove it.

Reply:
It's used in 2 titles but WebStorm marks it as unused :(